Tasks/AzureRmWebAppDeploymentV4/operations/KuduServiceUtility.ts

```diff
@@ -235,6 +253,27 @@ export class KuduServiceUtility {
         }
     }
 
+    private async _processDeploymentResponse(deploymentDetails: any): Promise<void> {
+        try {
+            var kuduDeploymentDetails = await this._appServiceKuduService.getDeploymentDetails(deploymentDetails.id);
+            tl.debug(`logs from ZIP deploy: ${kuduDeploymentDetails.log_url}`);
+
+            if(deploymentDetails.status == KUDU_DEPLOYMENT_CONSTANTS.FAILED || tl.getVariable('system.debug') && tl.getVariable('system.debug').toLowerCase() == 'true') {
+                await this._printZipDeployLogs(kuduDeploymentDetails.log_url);
+            }
+            else {
+                console.log(tl.loc('ZipDeployLogsURL', kuduDeploymentDetails.log_url));
```

Comment:
nit: for WAR deploy, the console statement statement will 'Zip Deploy logs' ?

Reply:
Done